src/main/java/ConsoleOut/ConsoleOut.java

```diff
@@ -0,0 +1,19 @@
+package ConsoleOut;
+import Main.History;
+
+public class ConsoleOut implements UserWriter {
+    private TextWriter textWriter;
+    private WaysWriter waysWriter;
+    private int delay;
+    private String color;
+    public ConsoleOut(int delay, String color) {
+        this.textWriter = new TextWriter();
+        this.waysWriter = new WaysWriter();
+        this.color= ColorCoder.getColorCode(color);
+        this.delay = delay;
+    }
+    public void writeHistory(History history){
+        textWriter.Write(history.getHistory(), delay, color);
+        waysWriter.Write(history.getWays(), delay, color);
+    }
+}
```

Comment:
This class has 19 lines of code, text writer has 15 and ways writer has 19. I think that this granularity is not good in here. Methods from text and ways writers can be in this class ( private methods).